src/AccountManagement.ts

```diff
@@ -185,7 +194,7 @@ export class AccountManagement {
   }
 
   /**
-   * Create an Account instance from an address. If no address is passed, the current Account is returned
+   * Create an Account instance from an address. If no address is passed, the current signing Account is returned
    */
   public getAccount(args?: { address: string }): Account {
```

Comment:
The problem is that you can use it to fetch any account, not just signing ones. We might want to separate this into

`getAccount(args: { address: string }): Account` and `getSigningAccount(): Account`

@VictorVicente @polymath-eric thoughts?

Reply:
I am not sure on the name since its overloaded, and the "fallback" value only gets the signing account.

This methods primary use case to me seems to be getting arbitrary account from addresses.

 I can see why the name change would make sense if getting the "current" account was the primary use case. But I don't think thats the case here.